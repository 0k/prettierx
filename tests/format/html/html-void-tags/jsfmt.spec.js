run_spec(__dirname, ["html", "angular", "vue"], { errors: { vue: true } });
run_spec(__dirname, ["html"], { htmlVoidTags: true, errors: { vue: true } });
run_spec(__dirname, ["angular", "vue"], {
  htmlVoidTags: true,
  errors: { vue: true },
});
