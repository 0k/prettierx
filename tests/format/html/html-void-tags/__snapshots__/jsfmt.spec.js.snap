// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input.html [vue] format 1`] = `
"Unexpected character \\"EOF\\" (2:1)
  1 | <input type='text'>
> 2 |
    | ^"
`;

exports[`input.html - {"htmlVoidTags":true} [vue] format 1`] = `
"Unexpected character \\"EOF\\" (2:1)
  1 | <input type='text'>
> 2 |
    | ^"
`;

exports[`input.html - {"htmlVoidTags":true} format 1`] = `
====================================options=====================================
htmlVoidTags: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text">

================================================================================
`;

exports[`input.html - {"htmlVoidTags":true} format 2`] = `
====================================options=====================================
htmlVoidTags: true
parsers: ["angular", "vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text" />

================================================================================
`;

exports[`input.html format 1`] = `
====================================options=====================================
parsers: ["html", "angular", "vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text" />

================================================================================
`;
